stl/inc/coroutine

```diff
@@ -0,0 +1,168 @@
+// coroutine standard header (core)
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _COROUTINE_
+#define _COROUTINE_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+
+#ifdef _RESUMABLE_FUNCTIONS_SUPPORTED
+#pragma message("The contents of <coroutine> are only available with standard coroutines. Use <experimental/coroutine> with /await.")

```

Comment:
clang-format is complaining about this line being more than 120 columns; you can fix it with `{path to MSVC repo}\src\tools\LLVM\LLVM10\x64\bin\clang-format -style=file -i {path of file(s) to format}`

Reply:
Reminder: you'll need to make changes here and in the internal PR to keep them synced. (Yes, we've been dealing with this hassle for every change in the last year =)).

Reply:
Thanks, should be better now (but I'm not super happy with the pragma message wrapping)